external_parser/unit_tests/test_metrics.cc

```diff
@@ -45,23 +43,22 @@ void should_match_expected_metrics(std::string infile_name,
       e.what()
     );
   }
-  remove(outfile.c_str());
 }
 
 BOOST_AUTO_TEST_CASE(metrics_increase_with_events_should_be_tracked) {
   std::string infile_name = "valid_joined_logs/average_reward_100_interactions.fb";
   std::map<std::string, int> expected_metrics = {
-    {"number_of_learned_events", 100},
-    {"number_of_skipped_events", 0}
+    {"total_learn_calls", 100},
```

Comment:
could it all be merged to the same code path? or is it not worth it

Reply:
all?